data/mships.txt

```diff
@@ -5719,7 +5719,7 @@ ship "CDB-12K Saturn"
 	explode "large explosion" 35
 	explode "huge explosion" 20
 	"final explode" "final explosion large"
-	description "A class of even larger dry bulk carrier than the Jupiter as exports and demand grew. Though due to lack of unloading gears it can only operate at more equipped ports. In addition to its' bulk bays it can also carry significant amount of container allowing captains to earn additonal profit."
+	description "A class of even larger dry bulk carriers than the Jupiter as exports and demand grew. Unfortunately, due to the lack of unloading gears, it can only operate at more specially-equipped ports. In addition to its' bulk bays it can also carry significant amount of containers, allowing captains to earn additional profit."
```

Comment:
```suggestion
	description "A class of even larger dry bulk carriers than the Jupiter as exports and demand grew. Unfortunately, due to the lack of unloading gears, it's operation is limited to fewer more well equipped ports. In addition to its' bulk bays it can also carry significant amount of containers, allowing captains to earn additional profit."
```